st2common/st2common/openapi.yaml

```diff
@@ -1190,6 +1190,91 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /api/exp/inquiries:
+    get:
+      operationId: st2api.controllers.exp.inquiries:inquiries_controller.get_all
+      description: Returns a list of all inquiries
+      parameters:
+        - name: limit
+          in: query
+          description: Number of executions to get
+          type: integer
+          default: 20
```

Comment:
Same as https://github.com/StackStorm/st2/pull/3653/files#r137388337, however, here it should be `100`. This is till under discussion: #3708